.github/workflows/rust-prs.yml

```diff
@@ -379,7 +379,6 @@ jobs:
           tool: nextest
 
       - name: Build CLI and Gateway
-        if: needs.what-changed.outputs.cargo-bin-specs
```

Comment:
Please let's comment this out instead, we have to remember restore it once extensions move out of this repo.